.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Python package
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: 
+      - master
+  schedule:
+  - cron: "0 4 * * *"
```

Comment:
What is the rationale behind the periodic scheduling? Is it actually needed?

Reply:
I should probably check if I need to restrict this to only `master` and eventually `tags`. The rationale is identifying breakage that might be due to external factors.